include/envoy/server/bootstrap_extension_config.h

```diff
@@ -15,6 +15,12 @@ namespace Server {
 class BootstrapExtension {
 public:
   virtual ~BootstrapExtension() = default;
+
+  /**
+   * Called when server is done initializing and we have the ServerFactoryContext available.
+   * @param context is the context to use for the extension.
+   */
+  virtual void onServerInitialized(Configuration::ServerFactoryContext& context) PURE;
```

Comment:
We can remove the parameter from this method, as the context will be same reference passed into createBootstrapExtension.